internal/runbits/refresh.go

```diff
@@ -29,9 +29,14 @@ func RefreshRuntime(
 	//if err != nil {
 	//	return locale.WrapError(err, "err_update_build_script")
 	//}
-	target := target.NewProjectTarget(proj, nil, trigger)
+	var t *target.ProjectTarget
```

Comment:
Can we simplify this to something like `target := target.NewProjectTarget(proj, commitIDFromProject(proj), trigger)` where `commitIDFromProject()` returns a `*strfmt.UUID`? I know the name probably doesn't make sense, but perhaps there is one that's better.

Single-character variable names are a bit icky and the duplicated `target.NewProjectTarget()` with one different argument does not seem ideal.